plugins/languages/otter.nix

```diff
@@ -0,0 +1,58 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+helpers.neovim-plugin.mkNeovimPlugin config {
+  name = "otter";
+  originalName = "otter.nvim";
+  defaultPackage = pkgs.vimPlugins.otter-nvim;
+
+  maintainers = [ lib.maintainers.perchun ];
+
+  settingsOptions = {
+    lsp = {
+      hover = {
+        border = helpers.defaultNullOpts.mkListOf lib.types.str [
+          "╭"
+          "─"
+          "╮"
+          "│"
+          "╯"
+          "─"
+          "╰"
+          "│"
+        ] "";
+      };
+      diagnostic_update_events = helpers.defaultNullOpts.mkListOf lib.types.str [ "BufWritePost" ] ''
+        `:h events` that cause the diagnostics to update. Set to:
+        [ "BufWritePost" "InsertLeave" "TextChanged" ] for less performant
```

Comment:
```suggestion
        `[ "BufWritePost" "InsertLeave" "TextChanged" ]` for less performant
```

This could also be done as an option example:

```nix
diagnostic_update_events = helpers.defaultNullOpts.mkListOf' {
  type = lib.types.str;
  pluginDefault = [ "BufWritePost" ];
  description = ''
    `:h events` that cause the diagnostics to update.

    See example for less performant but more instant diagnostic updates.
  '';
  example = [ "BufWritePost" "InsertLeave" "TextChanged" ];
};
```

Reply:
Your example is much better than upstream docs, thanks